cht-llama-cpp/models.py

```diff
@@ -1,12 +1,26 @@
+import json
 import multiprocessing
+from typing import Any, Dict, List
 
-from llama_cpp import Llama
+from llama_cpp import Llama, llama_chat_format, llama_types
 
 DEFAULT_N_THREADS = max(multiprocessing.cpu_count() // 2, 1)
 
 
+@llama_chat_format.register_chat_format("chatml")
+def initiate_chatml_prompt_template(
+    messages: List[llama_types.ChatCompletionRequestMessage],
+    **kwargs: Any,
+) -> llama_chat_format.ChatFormatterResponse:
+    # Until https://github.com/abetlen/llama-cpp-python/issues/717 supports ChatML.
```

Comment:
```suggestion
    # TODO: drop when https://github.com/abetlen/llama-cpp-python/issues/717 supports ChatML
```

Easier to find `TODO` strings